src/color-chart/color-chart.css

```diff
@@ -26,9 +26,18 @@
 	}
 }
 
+[part="color-channel"],
+slot[name="color-channel"]::slotted(*) {
```

Comment:
Can you add a comment? I couldn't tell what this was about and why we do this (granted, I just woke up, but still, it shouldn't need more thought than what I'm currently capable of 😀 )

Reply:
:) Makes sense. Done!